plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -23,3 +23,13 @@ if [ -f /usr/libexec/pk-command-not-found ]; then
         return $retval
     }
 fi
+
+# OSX command-not-found support
+# https://github.com/Homebrew/homebrew-command-not-found
+if [ -e /usr/local/bin/brew ]; then
+  if brew tap | command grep -q homebrew/command-not-found; then
```

Comment:
`brew command` is used to check whether a command is installed. If you want to hide `Error: Unknown command: command-not-found-init`, all you need to do is using `brew command command-not-found-init > /dev/null 2>&1`